README.md

```diff
@@ -25,15 +25,29 @@ This project uses a [lexer/parser](http://github.com/graphql-dotnet/parser) orig
 > If errors occur, it is recommended that you first check the behavior on the latest available preview version before
 > reporting a issue.
 
-You can install the latest version via [NuGet](https://www.nuget.org/packages/GraphQL/).
+You can install the latest version via NuGet.
 
+If you're targeting .NET Core 3+, you should use [GraphQL.SystemTextJson](https://www.nuget.org/packages/GraphQL.SystemTextJson/).
 ```
-> dotnet add package GraphQL
-> dotnet add package GraphQL -v 3.0.0-preview-1352
+> dotnet add package GraphQL.SystemTextJson
 ```
 
-Or you can get the latest pre-release packages from the [MyGet feed](https://www.myget.org/F/graphql-dotnet/api/v3/index.json).
+Else, you should use the legacy [GraphQL.NewtonsoftJson](https://www.nuget.org/packages/GraphQL.NewtonsoftJson/)
```

Comment:
```suggestion
Else, you should use the legacy and time-tested [GraphQL.NewtonsoftJson](https://www.nuget.org/packages/GraphQL.NewtonsoftJson/)
```

Reply:
Similar to above, I don't think we should be promoting this at all.

Reply:
ok